python/test/lib/test_json_arrays.py

```diff
@@ -1,5 +1,5 @@
 import unittest
-from data_extraction import json_arrays
+from lib import json_arrays
```

Comment:
This rename took me down a rabbit hole of Googling trying to find this `json_arrays` method that I thought was a Python library function.. turns out it wasn't and I should have looked at this commit in more detail first!

I think your intention was to bring the Rails' `lib` directory as somewhere to put code that isn't necessarily tied to the domain, and while I don't know the best practice of Python, perhaps we could think of a better name? The first result on Google regarding project structure pointed me to [The Hitchhiker's Guide to Python](http://docs.python-guide.org/en/latest/writing/structure/), which could suggest just a directory as 'helpers'?

Reply:
as far I could see they use helpers.py as a file to put helper thing in. I couldn't find any other standards as to where put this type of file in.